drones/image_processing/utils.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import cv2 as cv
+import imutils
+import drones.image_processing.normalization as normalization
+
+# These constants indicate color range in HSV of object to be detected.
+LOWER_BOUND_COLOR = np.array([0, 220, 100])
+UPPER_BOUND_COLOR = np.array([15, 255, 255])
+
+
+def detect_object(image: np.ndarray) -> int:
+    """Detect object in the image.
+
```

Comment:
this line is redundant. Also please change docstring to google format. We want to have same docstring style in whole project.

Reply:
This line is not redundant. Per PEP 257 - "Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.". Take a look [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) as well.